brain/compartmentReportView.h

```diff
@@ -78,6 +78,22 @@ class CompartmentReportView
      */
     BRAIN_API std::future<brion::Frames> load(double start, double end);
 
+    /** Load frames between start and end time stamps.
+     *
+     * @param start the start time stamp
+     * @param end the end time stamp
+     * @param stride the time stride
```

Comment:
If "stride" is not going to mean the same as stride in an array, then it should be called step. To me it would be confusing otherwise.